grails-app/controllers/org/pih/warehouse/UrlMappings.groovy

```diff
@@ -490,6 +490,11 @@ class UrlMappings {
             action = [GET: "invoiceTypeCodes"]
         }
 
+        "/api/invoiceItems"(parseRequest: true) {
+            controller = "invoiceApi"
+            action = [GET: "getAllInvoiceItems"]
```

Comment:
I think this should be in a separate `InvoiceItemApiController`, because what I see the `getInvoiceItems` in the `InvoiceAPIController` is meant to get the invoice items from a specific invoice item, so if this one is meant to pull all invoice items from different invoices, then this should land in a separate controller.

Reply:
we also have 
```groovy
"/api/invoiceItems/$id/validation" {
    controller = "invoiceApi"
    action = [POST: "validateInvoiceItem"]
}
```
so this will also have to end up in invoiceItemApicontroller

Reply:
Not really, I think this one can stay where it is, at least for now